From 408bded3befc625e16ef18f2ff5467e040b203e7 Mon Sep 17 00:00:00 2001
From: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Date: Wed, 27 May 2020 16:34:56 +0000
Subject: [PATCH] ListContainerHelper: Include <cstring> for memcpy()

IWYU. This fixes the libstdc++ build after commit 41989bd89 ("Remove/replace
unnecessary logging.h includes in .cc files (cc)").

Bug: 957519
Change-Id: I1e782e8310ca548947b2d541af6a2159d928d1f3
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2218090
Reviewed-by: vmpstr <vmpstr@chromium.org>
Commit-Queue: vmpstr <vmpstr@chromium.org>
Commit-Queue: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Auto-Submit: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Cr-Commit-Position: refs/heads/master@{#772267}
---

diff --git a/cc/base/list_container_helper.cc b/cc/base/list_container_helper.cc
index afd386e..7b594b4 100644
--- cc/base/list_container_helper.cc
+++ cc/base/list_container_helper.cc
@@ -7,6 +7,7 @@
 #include <stddef.h>
 
 #include <algorithm>
+#include <cstring>
 #include <vector>
 
 #include "base/check_op.h"
